module1/homework/grayscaleImages/test.cpp

```diff
@@ -4,8 +4,12 @@
 #include <vector>
 
 // TODO: include
+#include "compression.hpp"
 #include "gtest/gtest.h"
 
+const int height = 160;
+const int width = 240;
```

Comment:
jeżeli się skompiluje to zmienne mogą być constexpr

Reply:
i jeszcze przeniósłbym to do .hpp, bo .cpp będzie z niego czerpać, a nie z test.cpp